.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file is for your setup only --> gitignore. Because another person might be using another IDE, not vscode, or they have their own preferred setup.

Reply:
👍